python-sdk/nuscenes/utils/geometry_utils.py

```diff
@@ -141,3 +108,21 @@ def transform_matrix(translation: np.ndarray = np.array([0, 0, 0]),
         tm[:3, 3] = np.transpose(np.array(translation))
 
     return tm
+
+
+def quaternion_yaw(q: Quaternion) -> float:
+    """
+    Calculate the yaw angle from a quaternion.
+    Note that this only works for a quaternion that represents a box in lidar or global coordinate frame.
+    It does not work for a box in the camera frame.
```

Comment:
I like the move, but we need to do something about that this method "doesn't work in the camera frame". But we can address in subsequent PR I suppose.

Reply:
Or we should leave it here until we make it more general and can remove that statement.

Reply:
Yeah. Let's move it back for now.